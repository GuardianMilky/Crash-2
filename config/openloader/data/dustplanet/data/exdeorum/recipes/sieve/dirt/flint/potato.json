{"type":"exdeorum:sieve","ingredient":{"item":"minecraft:dirt"},"mesh":"exdeorum:flint_mesh","result":"minecraft:potato","result_amount":{"type":"minecraft:binomial","n":1.0,"p":0.13}}