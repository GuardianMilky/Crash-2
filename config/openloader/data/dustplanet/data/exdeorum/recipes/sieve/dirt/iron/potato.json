{"type":"exdeorum:sieve","ingredient":{"item":"minecraft:dirt"},"mesh":"exdeorum:iron_mesh","result":"minecraft:potato","result_amount":{"type":"minecraft:binomial","n":1.0,"p":0.15}}