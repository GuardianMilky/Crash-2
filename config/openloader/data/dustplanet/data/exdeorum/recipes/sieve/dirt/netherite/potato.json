{"type":"exdeorum:sieve","ingredient":{"item":"minecraft:dirt"},"mesh":"exdeorum:netherite_mesh","result":"minecraft:potato","result_amount":{"type":"minecraft:binomial","n":1.0,"p":0.3}}