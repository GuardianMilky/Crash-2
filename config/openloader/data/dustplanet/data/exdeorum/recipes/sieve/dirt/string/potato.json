{"type":"exdeorum:sieve","ingredient":{"item":"minecraft:dirt"},"mesh":"exdeorum:string_mesh","result":"minecraft:potato","result_amount":{"type":"minecraft:binomial","n":1.0,"p":0.1}}