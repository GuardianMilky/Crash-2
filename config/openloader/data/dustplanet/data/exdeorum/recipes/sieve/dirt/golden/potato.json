{"type":"exdeorum:sieve","ingredient":{"item":"minecraft:dirt"},"mesh":"exdeorum:golden_mesh","result":"minecraft:potato","result_amount":{"type":"minecraft:binomial","n":1.0,"p":0.175}}